app/javascript/src/Policies/components/PolicyConfig.jsx

```diff
@@ -18,13 +18,66 @@ type Props = {
 }
 
 const PolicyConfig = ({policy, actions}: Props) => {
-  return (<PoliciesForm
-    policy={policy}
-    submitForm={actions.submitPolicyConfig}
-    removePolicy={actions.removePolicyFromChain}
-    closePolicyConfig={actions.closePolicyConfig}
-    updatePolicy={actions.updatePolicyConfig}
-  />)
+  const { submitPolicyConfig, removePolicyFromChain, closePolicyConfig, updatePolicyConfig } = actions
+  const { humanName, version, summary, description, enabled, configuration, data, removable } = policy
+
+  const onSubmit = (policy) => {
+    return ({formData, schema}) => {
+      submitPolicyConfig({...policy, ...{configuration: schema, data: formData}})
+    }
+  }
+  const togglePolicy = (event) => {
+    updatePolicyConfig({...policy, ...{enabled: event.target.checked}})
+  }
+  const remove = () => removePolicyFromChain(policy)
+  const cancel = () => closePolicyConfig()
+
+  const isPolicyVisible = isNotApicastPolicy(policy)
+
+  return (
+    <section className="PolicyConfiguration">
+      <header>
+        <h2>Edit Policy</h2>
+        <HeaderButton type='cancel' onClick={cancel}>
+          Cancel
+        </HeaderButton>
+      </header>
+      <h2 className="PolicyConfiguration-name">{humanName}</h2>
+      <p className="PolicyConfiguration-version-and-summary">
+        <span className="PolicyConfiguration-version">{version}</span>
+        {' - '}
+        <span className="PolicyConfiguration-summary">{summary}</span>
+      </p>
+      <p className="PolicyConfiguration-description">{description}</p>
+      {isPolicyVisible &&
+      <label className="Policy-status" htmlFor="policy-enabled">
```

Comment:
```suggestion
        <label className="Policy-status" htmlFor="policy-enabled">
```
I think indentation is a bit short on lines 53, 63 and 73 right after the `... &&` predicate.